gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
```

Comment:
this got downgraded by an automated pr

Reply:
I've bumped the version in excavator so this shouldn't happen again.

Reply:
Actually, I've scratched that due to https://github.com/gradle/gradle/issues/24234.